### DISCUSSION TITLE
[질문] 여러 개의 Issues에 대한 pr을 한번에 보내도 되나요??

### DISCUSSION THREAD

**Discussion by boogi-woogi** [TEAM-DAME/Dame-Android#48] [2. Q&A] [answered]
![image](https://user-images.githubusercontent.com/88770426/201902490-31447c48-5cd1-4f46-8207-59c2f4ed7d3a.png)
현재 두 개의 이슈에 해당하는 것을 한번에 작업해버린 상태입니다. 
혹시 이렇게 여러 개의 issue에 해당하는 내용에 대해서 한번에 PR을 보내도 될까요?
그리고 혹시 가능하다면 pr보낼때 어떠한 형식으로 이슈를 어떻게 표현해야 할까요?
ex) (#num1, #num2)?

**Comment by onseok** [comment, marked as answer]
한 개의 pull request에 여러개의 issue를 연결하는 것도 가능합니다.
그러나 한 개의 pull request에 한 개의 issue를 연결하고자 한다면, 두 개중 하나의 이슈를 close 하고 나머지 하나의 이슈에 close 할 이슈의 작업 목록들을 추가하는 방법으로 진행하면 될 것 같습니다.
다만 두 기능이 합쳐졌으니 두 가지 기능을 포괄하는 이름으로 이슈 제목을 수정해주시면 좋을 것 같습니다. 
마찬가지로 pull request 보낼 때 연결할 이슈 번호는 나머지 하나의 이슈 번호를 작성하시면 될 것 같습니다.
*(pull request 제목은 close 된 pull request들을 참고하시면 금방 이해하실겁니다.)*

추가로, issue 마다 브랜치를 만드시고 각 issue에 해당하는 브랜치에서 작업을 하시는것을 추천드립니다.
예를 들어, 해당 사진에서는 `feature/boogi-woogi/init-notification-logic` 브랜치와 `feature/boogi-woogi/init-notification-layout` 브랜치 2가지로 작업하신 후, 각각 pr을 보내시는 것이 더 좋을 것 같습니다.

**Reply by boogi-woogi** [reply]
넵 이해했습니다!
